dolphin/db/sqlalchemy/api.py

```diff
@@ -121,3 +129,10 @@ def storage_get(storage_id):
         .filter(Storage.id == storage_id) \
         .first()
     return storage_by_id
+
+
+def storage_get_all():
+    this_session = get_session()
+    this_session.begin()
```

Comment:
Will this take care of closing the session or handling any exception or rollback?

Reply:
traditional approach is to use " with this_session.begin(): ......"

Reply:
yes , Did some analysis on this 'with' i smaily used when multiple tarnsactions are there in one session.. 
Once yu come out of the scope of session, session will be closed.. exception can be caught and    rollback can be called on session..
Will improve this for update queries.. thanks for suggestion..

Reply:
ok